gcloud/datastore/connection.py

```diff
@@ -493,8 +508,8 @@ def delete_entities(self, dataset_id, key_pbs):
         :type key_pbs: list of :class:`gcloud.datastore.datastore_v1_pb2.Key`
         :param key_pbs: The keys to delete from the datastore.
 
-        :rtype: boolean
-        :returns: `True`
+        :rtype: :class:`bool`
```

Comment:
Good point. I think the right move is just using "string", "boolean", "integer" and hoping we stay consistent.

Will try to find them all :S